index.html

```diff
@@ -2141,6 +2173,268 @@ <h2>
         This section is a work in progress as this document evolves.
       </p>
     </section>
+    <section id="user-agent-automation" data-cite="webdriver">
+      <h2>
+        User Agent Automation
+      </h2>
+      <p>
+        For the purposes of user agent automation and web application testing,
+        this document defines a number of [[WebDriver]] [=extension commands=].
+      </p>
+      <section id="webdriver-extension-capability">
+        <h3>
+          Digital Credentials WebDriver Extension Capability
+        </h3>
+        <p>
+          In order to advertise the availability of [=extension commands=]
+          defined below, a new extension capability is defined.
+        </p>
+        <table class="data">
+          <thead>
+            <tr>
+              <th>
+                Capability
+              </th>
+              <th>
+                Key
+              </th>
+              <th>
+                Value Type
+              </th>
+              <th>
+                Description
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                Virtual Wallet Support
```

Comment:
```suggestion
                Virtual Credential Manager Support
```

Reply:
We haven't really used "wallet" formally... we've stuck to holder... so I wonder if this should be "virtual holder"

Reply:
I think "virtual (whatever)" is not accurate and will be misunderstood.

Better might be "software (whatever)" (even if it's implemented on hardware, it's still software doing the lifting).

Rather than `Virtual Credential Manager Support`, I would suggest `Support for Software Credential Managers` or perhaps `Support for Agentic Credential Managers`.